scripts/libperf.sh

```diff
@@ -0,0 +1,5 @@
+download_and_extract https://github.com/NT-Bourgeois-Iridescence-Technologies/PerfLib/archive/master.zip PerfLib
+make
+mkdir -p $(psp-config --psp-prefix)/include/
+cp -v include/*.h $(psp-config --psp-prefix)/include/
```

Comment:
Since perflib has only one header file, we can be more specific.

Reply:
Done